src/Microsoft.ML.TimeSeries/STL/BasicParameters.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Microsoft.ML.TimeSeries
+{
+    public class BasicParameters
```

Comment:
Can you make this class `internal`? It seems that it's only used inside the constructor of `FastLoess `and `Loess`, and so there's no reason to expose them to the user.

Reply:
In ML.NET we have a commitment with our users to not make any public API breaking changes. This means that whenever we introduce a public class or method, we won't be able to rename it or remove it in the future, and adding things like parameters to them becomes tricky. So in general we prefer to expose as minimum as possible as `public`, and we do prefer to use the `internal` access modifier to avoid these potential problems.

---
In reply to: [440495082](https://github.com/dotnet/machinelearning/pull/5202#discussion_r440495082) [](ancestors = 440495082)

Reply:
Yes, updated.

---
In reply to: [440495855](https://github.com/dotnet/machinelearning/pull/5202#discussion_r440495855) [](ancestors = 440495855,440495082)